catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/ResourceCache.java

```diff
@@ -268,9 +270,9 @@ public void put(ReliableResource reliableResource) {
     @Override
     public void removePendingCacheEntry(String cacheKey) {
         if (!pendingCache.remove(cacheKey)) {
-            LOGGER.debug("Did not find pending cache entry with key = {}", cacheKey);
+            LOGGER.info("Did not find pending cache entry with key = {}", cacheKey);
```

Comment:
Why did we change the log level here? This is going to flood the log under high load and cause performance issues.